ndarray/ndarray-tfjs/src/jsMain/kotlin/io.kinference.ndarray/extensions/MemoryExtension.kt

```diff
@@ -2,12 +2,23 @@ package io.kinference.ndarray.extensions
 
 import io.kinference.ndarray.arrays.ArrayTFJS
 import io.kinference.ndarray.arrays.NDArrayTFJS
-import io.kinference.ndarray.core.tidy
+import io.kinference.ndarray.core.*
 
-fun tidy(fn: () -> Array<ArrayTFJS>): Array<ArrayTFJS> = tidy(fn, null)
+suspend fun tidy(fn: suspend () -> Array<ArrayTFJS>): Array<ArrayTFJS> {
```

Comment:
So, what does that thing do?

Reply:
That's suspendable version of TFJS Tidy, [link](https://js.tensorflow.org/api/latest/#tidy)

Reply:
In general: automatic memory cleaner